File: src/go/vfs_test.go
package main

import (
	"bufio"
	"fmt"
	"os"
)

func main() {
	fmt.Println("--- VFS Test ---")

	filesToTest := []string{"/test.txt", "test.txt", "./test.txt"}

	for _, path := range filesToTest {
		fmt.Printf("Testing path: %s\n", path)
		file, err := os.Open(path)
		if err == nil {
			scanner := bufio.NewScanner(file)
			if scanner.Scan() {
				fmt.Printf("Successfully read: %s\n", scanner.Text())
			} else {
				fmt.Println("File is empty or could not read line.")
			}
			file.Close()
		} else {
			fmt.Printf("Failed to open %s\n", path)
		}
	}
}

File: src/python/vfs_test.py
def main():
    print("--- VFS Test ---")
    
    files_to_test = ["/test.txt", "test.txt", "./test.txt"]
    
    for path in files_to_test:
        print(f"Testing path: {path}")
        try:
            with open(path, 'r') as file:
                line = file.readline().rstrip('\n')
                if line:
                    print(f"Successfully read: {line}")
                else:
                    print("File is empty or could not read line.")
        except Exception:
            print(f"Failed to open {path}")

if __name__ == "__main__":
    main()
